index.html

```diff
@@ -1,1642 +1,246 @@
-            </td>
-          </tr>	
-          <tr>
-            <th scope="row"><code>data</code></th>
-            <td>String</td>
-            <td></td>
-			<td>No</td>
-            <td>
-              <span>Attribute set for components to facilitate data storage and reading.</span>
-            </td>
-          </tr>		  
-		</tbody>
-	  </table>
-     <p>The 'refresh' component not support additional events except the common events.</p>	  
-	</section>	
-</section>
-  <section id="sec-basic-components">
-      <h2>Basic Components</h2>
-      <section>
-      <h3>image</h3>
-	  <p>The 'image' component is used to render and display images.</p>
-	  <p>In addition to common attributes, the 'image' component supports the following attributes:</p>
+  <section id="sec-scripting">
+    <h2>Scripting Resources</h2>
+    <p>
+      MiniApp user agents support scripting resources to define the business logic and operation of the components. Although the aim of this specification is not to recommend a specific scripting language, the document considers ECMAScript (broadly known as JavaScript) [[JAVASCRIPT]] as the scripting language by default.
+    </p>
+    <p>
+      As indicated in [[MINIAPP-PACKAGING]], each MiniApp component has a scripting file (with the `.js` extension). Since each component has its own scope, the functions and variables defined in the scripting file have global scope across the entire component, but they are isolated from the rest of the MiniApp components.     
+    </p>
+    <p>
+      Scripts in components support module management. Modules have an independent scope, so the elements defined in a module are private by default and invisible to other modules. The exposition of the concrete module variables or functions MUST be explicitly made through the export declarations. Only local modules (i.e., hosted in the same package) MAY be imported.
+    </p>
+    <p data-cite="HTML">
+      HTML resources (with the .html extension) MAY import local scripts, such as auxiliary libraries shared by the MiniApp components. These script documents SHOULD be hosted in the `/common` directory as defined in [[MINIAPP-PACKAGING]], so they will be available to any component in the package. As in the case of scripting modules, only local scripts (i.e., hosted in the same package) will be accessible to the components. Developers MAY include any scripting resource hosted in the same package from the HTML, using the `script` element, equivalent to [^script^] in HTML, with the [^script/src^] attribute to specify the URI of the document to import. The functions and variables of the imported script will have a limited scope across the entire component.
+    </p>
+    <pre class="example" title="Scripting modules import"><code>//Static import
+import utils from '../Common/utils.js'
+
+// Dynamically
+const utils = require('/Common/utils.js')

```

Comment:
What about something like:

```js
const utils = await import('/Common/utils.js');
```

Reply:
It makes sense, since [it's standard](https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-import-calls). 

Added at b9c8ffab1938d52eb06e844d2a3758e4f26a10d1